src/test/java/uk/gov/di/ipv/cri/common/library/util/SignedJWTFactoryTest.java

```diff
@@ -54,6 +54,32 @@ void shouldCreateASignedJwtSuccessfullyFromJWTClaimsSetString()
         assertThat(signedJWT.verify(new ECDSAVerifier(ECKey.parse(EC_PUBLIC_JWK_1))), is(true));
     }
 
+    @Test
```

Comment:
Can we add a test that asserts the structure of the keyId i.e
keyId.split(":")
[0]=did
[1]=web
[2]=issuer
[3]=mocked-value(hashed)